packages/jellyfish-api-core/src/category/loan.ts

```diff
@@ -0,0 +1,37 @@
+import { ApiClient } from '../.'
+
+/**
+ * Loan RPCs for DeFi Blockchain
+ */
+export class Loan {
+  private readonly client: ApiClient
+
+  constructor (client: ApiClient) {
+    this.client = client
+  }
+
+  /**
+   * Destroys a loan scheme.
+   *
+   * @param {string} id Unique identifier of the loan scheme, max 8 chars
+   * @param {number} [activateAfterBlock] Block height at which new changes take effect
+   * @param {DeleteLoanSchemeOptions} [options]
+   * @param {UTXO[]} [options.utxos = []] Specific UTXOs to spend
+   * @param {string} [options.utxos.txid] Transaction Id
+   * @param {number} [options.utxos.vout] Output number
+   * @return {Promise<string>} Hex string of the transaction
+   */
+  async destroyLoanScheme (id: string, activateAfterBlock?: number, options: DeleteLoanSchemeOptions = {}): Promise<string> {
```

Comment:
why not use `utxos: UTXO[]` directly  instead of `options: DeleteLoanSchemeOptions` here. can avoid additional  interface.
```suggestion
  async destroyLoanScheme (id: string, activateAfterBlock?: number, utxos: UTXO[] = []): Promise<string> {
```

Reply:
See guideline at : https://github.com/DeFiCh/jellyfish/blob/main/CONTRIBUTING.md#exported-functions-max-2-args-put-the-rest-into-an-options-object

Reply:
Agree that creating an interface just for utxos might create more confusion

Reply:
I wrote what you suggested here a few months ago, But Fuxing said it is better to stick with  third parameter is always an option